### PR TITLE
User profile 2

### PR DESCRIPTION
**Pull request by trangkarol**

### CODE REVIEW

**Comment by trangkarol** [comment]
screenshot from 2017-10-25 17-48-07

**Comment by trangkarol** [comment]
ready

File: app/views/friends/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<%= render "users/header" %>
+<div class="container-stories">
+  <div class="container">
+    <%= render "users/left_bar" %>
+    <div class="col-md-9" id="div-show-user">
+      <% @list_friends.each do |member| %>
```
Comment (dothidiemthao) [inline app/views/friends/index.html.erb]: Dùng render partial nhé

**Comment by dothidiemthao** [comment]
commented

**Comment by trangkarol** [comment]
ready

**Pull request merged**